src/aiperf/config/flags/cli_config.py

```
@@ -518,6 +518,19 @@ def url(self) -> str:
+    dataset_filters: Annotated[
+        list[str],
+        Field(
+            description="Dataset-specific filter in key=value form. Repeat for multiple "
+            "filters. Only supported by public datasets that declare filter support.",
+        ),
+        CLIParameter(
+            name=("--dataset-filter",),
+            consume_multiple=True,
+            group=Groups.INPUT,
+        ),
+    ] = []
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)